### ISSUE TITLE
Can't build project

### ISSUE THREAD

**Issue by robertjf** [KOBENDigital/Iconic#12]
Hey @skartknet,

Went to build the project so I could debug an issue with using Iconic with Stacked/Nested Content (separate issue report coming) but the solution file is missing from the project; and the project fails to build with the following error:

> This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is ..\packages\BuildWebCompiler.1.11.326\build\BuildWebCompiler.targets.

To fix this I had to change all the ..\packages references in the csproj file to packages (removing the `..\`)

Can you please check in the correct .sln file and anything else that may be missing?

**Comment by robertjf** [comment]
Also, the post-build script fails with the following error:

> The command "xcopy /y/s "C:\Users\RobertFoster_lxyercz\source\repos\Iconic\\App_Plugins" "C:\Users\RobertFoster_lxyercz\source\repos\Iconic\\Umbraco79\App_Plugins"" exited with code 2.

**Comment by skartknet** [comment]
Hey Rob, I just included the actual Iconic project instead the testing environment. I have included tht whole solution now. Let me know how you go. Cheers.

**Comment by robertjf** [comment]
much better, thanks 💯 

Are you able to document the sample website username/password?

**Comment by robertjf** [comment]
And maybe move the Readme.md file down to the root of the solution so it automatically comes up - there are some broken asset links in it too, it appears

**Comment by skartknet** [comment]
Yeah, I'll have to clean the solution. That project was for my own testing, I didn't intend to include it in the repo.
Credentials:
Username: mario@koben.com.au
password: 123123123123
